test/Microsoft.DocAsTest.Test/JsonDiffTest.cs

```diff
@@ -36,10 +36,7 @@ public class JsonDiffTest
         [InlineData("{'a': 1, 'b': { 'c': 2 }}", "{'a': 1, 'b': { 'c': 3 }}", @"
 -    'c': 2
 +    'c': 3")]
-        [InlineData("{'a': 1}", "{'a': 1, 'b': 2}", @"
--  'a': 1
-+  'a': 1,
-+  'b': 2")]
+        [InlineData("{'a': 1}", "{'a': 1, 'b': 2}", "")]
```

Comment:
shall we have some tests for `JsonDiffBuilder` without `UseAdditionalProperties`?
Or can we construct the tests into different ones, each test against one `Use` extension method? That would be easier to see what is failing as well

Reply:
yeah, that's better